WcaOnRails/app/models/competition.rb

```diff
@@ -43,6 +43,12 @@ def must_have_at_least_one_event
       errors.add(:eventSpecs, "Competition must have at least one event")
     end
   end
+  validate :must_have_at_least_one_delegate, if: :isConfirmed?
+  def must_have_at_least_one_delegate
+    if delegate_ids.length == 0
+      errors.add(:delegates, "Competition must have at least one WCA delegate")
```

Comment:
Indeed I do.
Why is there a delegates field? lol

I tried doing the same for events, but there is no f.input for them, just a label.